runtimes/alibi-detect/tests/conftest.py

```diff
@@ -125,6 +127,21 @@ def drift_detector_settings(
     )
 
 
+@pytest.fixture
+def online_drift_detector_settings(
+    online_drift_detector_uri: str,
+) -> ModelSettings:
+    return ModelSettings(
+        name="alibi-detect-model",
+        implementation=AlibiDetectRuntime,
+        parameters=ModelParameters(
+            uri=online_drift_detector_uri,
+            version="v1.2.3",  # TODO - should this match version.py?
```

Comment:
Question: what version should this be?

Reply:
This version is arbitrary (it can be anything the user wants). We could even set it to `None`.